src/Process/Listener/FileLoggerSubscriber/BaseFileLoggerSubscriber.php

```diff
@@ -73,7 +75,9 @@ public function onMutationTestingFinished(MutationTestingFinished $event)
         $logTypes = $this->filterLogTypes($logTypes);
 
         foreach ($logTypes as $logType => $directory) {
-            $this->useLogger($logType);
+            if ($this->logVerbosity !== LogVerbosity::NONE) {
```

Comment:
I'd say that this condition can be moved to the top of this method and do smth like:

```
public function onMutationTestingFinished(MutationTestingFinished $event)
    {
         if ($this->logVerbosity !== LogVerbosity::NONE) {
             return false;
         }

        $logTypes = $this->infectionConfig->getLogsTypes();

        if (empty($logTypes)) {
            return;
        }

        $logTypes = $this->filterLogTypes($logTypes);

        foreach ($logTypes as $logType => $directory) {
            $this->useLogger($logType);
        }
    }
```

Reply:
👍 except for the use of `empty()` :)